docs/next.config.js

```diff
@@ -124,52 +156,6 @@ module.exports = {
               },
             ],
           },
-          // transpile 3rd party packages with dependencies in this repository
-          {
-            test: /\.(js|mjs|jsx)$/,
-            resourceQuery: { not: [/raw/] },
-            include:
-              /node_modules(\/|\\)(notistack|@mui(\/|\\)x-data-grid|@mui(\/|\\)x-data-grid-pro|@mui(\/|\\)x-license-pro|@mui(\/|\\)x-data-grid-generator|@mui(\/|\\)x-date-pickers-pro|@mui(\/|\\)x-date-pickers)/,
```

Comment:
We will need to test that notistack and x-data-grid are using the local version of Material UI.

Reply:
Seems to work for `x-data-grid` (because [I left the externals in](https://github.com/mui/material-ui/blob/f1a6e3fb0368d9e854569bd01f21ea5e52f9fe5f/docs/next.config.js#L89)). Inspected `notistack`, the version installed doesn't seem to be importing any `@mui/` packages anywhere. It added us in `peerDependencies` for some reason though.

https://user-images.githubusercontent.com/2109932/174404956-12397ff7-9c75-4180-899e-a15adaa59e26.mov